Jenkinsfile

```
@@ -315,10 +373,19 @@ stage('Unit Test') {
             init_git()
             unpack_lib('gpu', tvm_multilib)
             timeout(time: max_time, unit: 'MINUTES') {
-              sh "${docker_run} ${ci_gpu} ./tests/scripts/task_ci_setup.sh"
-              sh "${docker_run} ${ci_gpu} ./tests/scripts/task_sphinx_precheck.sh"
-              sh "${docker_run} ${ci_gpu} ./tests/scripts/task_python_unittest_gpuonly.sh"
-              sh "${docker_run} ${ci_gpu} ./tests/scripts/task_python_integration_gpuonly.sh"
+              ci_setup(ci_gpu)
+              sh (
+                script: "${docker_run} ${ci_gpu} ./tests/scripts/task_sphinx_precheck.sh",
+                label: "Check Sphinx warnings in docs",
+              )
+              sh (
+                script: "${docker_run} ${ci_gpu} ./tests/scripts/task_python_unittest_gpuonly.sh",
+                label: "Run GPU unit tests",
```

If these are specifically the Python tests and not other languages, should we label them so?

```suggestion
                label: "Run Python GPU unit tests",
```